packages/dataviews/src/style.scss

```diff
@@ -1,12 +1,12 @@
 .dataviews-wrapper {
-	width: 100%;
 	height: 100%;
 	overflow: auto;
 	box-sizing: border-box;
 	scroll-padding-bottom: $grid-unit-80;
 }
 
 .dataviews-filters__view-actions {
+	box-sizing: border-box;
```

Comment:
This style is present in WordPress "reset.css" but is needed for places like storybook or usage outside of WP. This makes the style more portable.